packages/react-ui/components/Kebab/__stories__/Kebab.stories.tsx

```diff
@@ -130,6 +134,37 @@ WithFixedMenuHeight.story = { name: 'With fixed menu height', parameters: { cree
 export const KebabWithoutAnimations = () => <SomethingWithKebab disableAnimations size="small" />;
 KebabWithoutAnimations.story = { name: 'Kebab without animations', parameters: { creevey: { skip: [true] } } };
 
+export const ProgrammaticFocus = () => {
+  let kebab: Nullable<Kebab>;
+
+  function focus() {
+    if (kebab) {
+      kebab.focus();
+    }
+  }
+
+  function blur() {
+    if (kebab) {
+      kebab.blur();
+    }
+  }
+
+  return (
+    <div style={{ width: 200, textAlign: 'center' }}>
+      <Gapped>
+        <button onClick={focus}>Focus</button>
+        <button onClick={blur}>Blur</button>
+        <Kebab
+          size="small"
+          ref={el => (kebab = el)}
+        />
+      </Gapped>
+    </div>
+  );
+};
+
+ProgrammaticFocus.story = { name: ' Kebab with programmatic focus', parameters: { creevey: { skip: [true] } } };
```

Comment:
```suggestion
ProgrammaticFocus.story = { name: 'Kebab with programmatic focus', parameters: { creevey: { skip: [true] } } };
```
---
Почему заскипал? Хороший тест получится.

Ещё можно сделать тест на `jest` для пропов `onFocus` и `onBlur`.